core/src/main/resources/hudson/util/HudsonIsRestarting/index.jelly

```diff
@@ -60,8 +58,7 @@ THE SOFTWARE.
                     </div>
                 </div>
             </div>
-            <script src="${resURL}/scripts/loading.js" type="text/javascript"></script>
-            <script>safeRedirector(window.location.href);</script>
+            <script src="${resURL}/scripts/loading.js" type="text/javascript" />
```

Comment:
@Wadeck last time I looked at self-closing `script` tag, it wasn't supported by all web browsers. I know, makes no sense (html != xml), but just to be sure, did you test it? 

First comment about that I can find is from 2013... but basically it would be because `script` can have a `src` and code in its body. So the closing tag seems to be mandatory.

This might also apply to `div` you have on line 50 as well.

Reply:
It would be an invalid script tag in Jelly without a closing tag or self close. I'm not sure how Jelly outputs these types of tags in HTML, though.

Reply:
After your comment I looked for `<script.*/>` and found 18 matches in jenkins-core. If it was not working, the non-supporting browser will already have troubles :) (and +25 for div)

I can change them if you're not confident with the existing code.